turbo.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Tempted to say we don't need this file, it's for a "turborepo"? No idea what that is but theme is anything we don't need we should not check in

Reply:
Yes, should be removed.